docs/content/how-to/disaster-recovery/scenarios/cross-cluster-migration.md

```
@@ -138,14 +120,36 @@ hypershift create oadp-backup \
 
 **Or using a manual Velero Backup manifest** — see the [OADP method reference](../methods/oadp.md) for platform-specific backup manifests.
 
-#### Step 3: Verify Backup Completion
+#### Step 2: Verify Backup Completion
 
 ```bash
```

markdownlint-cli2 (0.23.2)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)